src/avfoundation.cs

```diff
@@ -2492,9 +2768,15 @@ interface AVAsset : NSCopying {
 		[Export ("tracksWithMediaType:")]
 		AVAssetTrack [] TracksWithMediaType (string mediaType);
 
+		[Wrap ("TracksWithMediaType (mediaType.GetConstant ())")]
+		AVAssetTrack [] TracksWithMediaType (AVMediaTypes mediaType);
+
 		[Export ("tracksWithMediaCharacteristic:")]
 		AVAssetTrack [] TracksWithMediaCharacteristic (string mediaCharacteristic);
 
+		[Wrap ("TracksWithMediaType (mediaCharacteristic.GetConstant ())")]
+		AVAssetTrack [] TracksWithMediaCharacteristic (AVMediaCharacteristics mediaCharacteristic);
```

Comment:
This one is using a smart enum, isn't it? I guess you mean to update all the TraksWith* to use smart enums, correct?